docs/content/topics/contributor-guide/ladder.md

```diff
@@ -0,0 +1,159 @@
+> projects or source code repositories owned by the __@brigadecore__ org.
+
+## Community Participant
+
+__Community Participant__ is the _de facto_ role for any Brigade user who ceases
+to be anonymous and begins publicly interacting with the Brigade Community. Some
+ways to become a Community Participant include, but are not limited to,
+voluntarily:
+
+* Participating in Community discussions (e.g. on Slack)
+* Assisting other Community Participants
+* Opening or commenting on issues
+* Commenting on PRs
+* Trying out new releases
+* Attending Community events
+
+All Community Participants must adhere to the
+[CNCF Code of Conduct](https://github.com/cncf/foundation/blob/master/code-of-conduct.md)
+
+## Contributor
+
+A __Contributor__ is a Community Participant who goes a step farther and
+contributes _directly_ to moving the Brigade Project forward in some fashion.
+All that is required to become a Contributor is to start contributing!
+_Contributions do not need not be code._ Some ways to contribute include, but
+are not limited to, voluntarily:
+
+* Opening PRs containing code, documentation, or other content
+  * Such contributions should adhere to our [Contributor Guide](./index.md)
+* Using knowledge of Brigade to substantially assist other Community
+  Participants
+* Publicly promoting the Brigade Project
+
+## Organization Member
+
+An __Organization Member__ is an established and trusted Contributor who
+contributes _regularly_ to the Brigade Project and has been granted membership
+by the Core Maintainers in the __@brigadecore__ GitHub org.
+
+An Organization Member must have:
+
+* Made _several_ contributions to the Project
+* Earned the trust of the Core Maintainers
+* A _demonstrated need_ for org membership
+  * Usually this will be because Core Maintainers have determined that the
+    individual requires (and is trusted with) elevated permissions _vis-à-vis_
+    one or more GitHub repositories belonging to the __@brigadecore__ GitHub
+    org.
+
+Privileges include:
+
+* CI processes that run automatically for their PRs
+* Authorizing CI processes to run for PRs from other Contributors
+
+## Project Maintainer
+
+Project Maintainers are established and trusted Contributors who are granted
+broad responsibility for one or more, but not all, projects owned by the
+__@brigadecore__ GitHub org. They are, generally, but not necessarily,
+Organization Members. Project Maintainers (along with Core Maintainers) have
+responsibility for ensuring the quality of code contributions by means of
+conducting (or delegating) thorough reviews of PRs.
+
+Project Maintainers must:
+
+* Have earned the trust of the Core Maintainers
+* Exercise sound judgement for the good of their project(s) and the Project,
+  independent of their employer's interests
+* Contribute regularly to their project(s)
+* Conduct PR reviews
+* Mentor other Contributors
+* Regularly participate in Community meetings
+
+Privileges include:
```

Comment:
Do you want to mention that maintainers/core maintainers have write access to the repository/org?

Reply:
Yes. That's conspicuously absent. Thanks for catching that.